app/main.py

```diff
@@ -1,34 +1,76 @@
 class Deck:
-    def __init__(self, row, column, is_alive=True):
-        pass
+    def __init__(self, row: int, column: int, is_alive: bool = True) -> None:
+        self.row = row
+        self.column = column
+        self.is_alive = is_alive
+
+    def __getitem__(self, item: int) -> int:
+        if item == 0:
+            return self.row
+        elif item == 1:
+            return self.column
 
 
 class Ship:
-    def __init__(self, start, end, is_drowned=False):
-        # Create decks and save them to a list `self.decks`
-        pass
+    def __init__(
+            self,
+            start: tuple,
+            end: tuple,
+            is_drowned: bool = False
```

Comment:
You could annotate optional arguments:
```suggestion
            is_drowned: bool | None = False
```